src/components/20-molecules/datepicker/index.js

```diff
@@ -19,6 +19,7 @@ export class Datepicker extends LitElement {
 
   static get properties() {
     return {
+      'data-test-id': { type: String, reflect: true },
```

Comment:
why reflect? we dont need to change it to outside

Reply:
It's not passed down to the react component without, and then the smoke test fails.

Reply:
weird as reflect just makes sure that changes inside the component are synced up to the attr.. pretty sure u don need it